.cypress/cypress/integration/northamptonshire.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * These don't run by default, need to run the tests with
+ * ./bin/browser-tests run --cobrand northamptonshire --area_id 2234 --coords '52.236251,0.892052
+ *
+ * NB: most of the non northamptonshire tests will then fail
+ */
+
+var cobrand = Cypress.env('cobrand');
+var only_or_skip = (cobrand == 'northamptonshire') ? describe.only : describe.skip;
+
+only_or_skip('Northamptonshire cobrand specific testing', function() {
+
+    it('loads the right front page', function() {
+        cy.visit('/');
+        cy.contains('Northamptonshire');
+    });
+
+    it('prevents clicking unless asset selected', function() {
+      cy.server();
+      cy.fixture('trees.json');
+      cy.fixture('trees_none.json');
+      cy.route('**/render-layer/**', 'fixture:trees_none.json').as('empty-trees-layer');
+      cy.route('**/16463/10787**', 'fixture:trees.json').as('trees-layer');
+      cy.route('/report/new/ajax*').as('report-ajax');
+      cy.visit('report/new?longitude=0.892052&latitude=52.236251');
```

Comment:
Nice! Perhaps one of these could go to /around and click the map, or did that prove too tricky?